plugins/builtin/strategy/target-value/plugin/plugin.go

```diff
@@ -15,6 +15,14 @@ const (
 	// pluginName is the unique name of the this plugin amongst strategy
 	// plugins.
 	pluginName = "target-value"
+
+	// These are the keys read from the RunRequest.Config map.
+	runConfigKeyTarget    = "target"
+	runConfigKeyPrecision = "precision"
```

Comment:
bike-shedding: i'm not a gigantic fan of `precision`... i think `threshold` might be more explicit: we require the factor to exceed the threshold. `precision` suggests to me that we're just being sloppy with our arithmetic.

Reply:
Yeah, that's not a good name. I started out thinking about in terms of significant digits, but that was kind of hard to reason about and slightly more complicated to calculate, so I change to something like an error interval, but calling it `error` sounded like a bad idea. Threshold makes sense.